website/frontend/src/components/blocks/SearchForm/SearchForm.tsx

```diff
@@ -1,60 +1,49 @@
-import React, { VFC } from 'react';
+import React from 'react';
+import { Button } from '@gear-js/ui';
 import { Field, Form, Formik } from 'formik';
 import { SearchModel } from 'types/program';
 import { SearchIcon } from 'assets/Icons';
+import searchIcon from 'assets/images/search.svg';
 import { Schema } from './Schema';
-import './SearchForm.scss';
+import styles from './SearchForm.module.scss';
 
 type Props = {
-  handleSearch: (searchQuery: string) => void;
-  handleRemoveQuery: () => void;
+  term: string;
   placeholder: string;
+  handleSearch: (term: string) => void;
 };
 
-export const SearchForm: VFC<Props> = ({ handleSearch, handleRemoveQuery, placeholder }) => {
-  const mapInitialValues = () => ({
-    searchQuery: '',
-  });
-
+const SearchForm = ({ term, placeholder, handleSearch }: Props) => {
   return (
     <Formik
-      initialValues={mapInitialValues()}
+      initialValues={{ term }}
       validationSchema={Schema}
       validateOnBlur
       onSubmit={(values: SearchModel) => {
-        handleSearch(values.searchQuery);
+        handleSearch(values.term);
       }}
       onReset={() => {
-        handleRemoveQuery();
+        handleSearch('');
       }}
     >
       {() => (
         <Form>
-          <div className="search-form">
-            <div className="search-form--field-wrapper">
-              <div className="search-form__icon">
+          <div className={styles.searchForm}>
+            <div className={styles.inputWrapper}>
+              <div className={styles.iconWrapper}>
                 <SearchIcon color="#BBBBBB" />
               </div>
```

Comment:
If you want to refactor markup in this PR, we have `Input` component in the `@gear-js/ui`

Reply:
Does it work with Formik?

Reply:
Yes

Reply:
I haven't tested it yet, but it should